pkg/cmd/chat/channel/channel.go

```diff
@@ -0,0 +1,230 @@
+package channel
+
+import (
+	"encoding/json"
+	"errors"
+	"text/tabwriter"
+	"time"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/cheynewallace/tabby"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{
+		getCmd(),
+		createCmd(),
+		deleteCmd(),
+		updateCmd(),
+		listCmd()}
+}
+
+func getCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get-channel --type [channel-type] --id [channel-id]",
```

Comment:
I would ask for `cid`, one flag is easier

Reply:
maybe but the rest of the commands use type and id, so I wanted to have it consistent